templates/items/tool.html

```diff
@@ -15,6 +15,9 @@ <h4 class="item-type">{{itemType}}</h4>
             </div>
 
             <ul class="summary flexrow">
+                <li>
+                    {{lookup config.toolTypes data.toolType }}
+                </li>
```

Comment:
Rather than leaving a blank space for no tool, you can just revert to the default `"Tool"` label:

```suggestion
                <li>
                    {{#if data.toolType}}
                      {{lookup config.toolTypes data.toolType}}
                    {{else}}
                      {{localize "DND5E.ItemTypeTool"}}
                    {{/if}}
                </li>
```